deepwell/src/services/site_member/structs.rs

```diff
@@ -0,0 +1,36 @@
+/*
+ * services/site_member/structs.rs
+ *
+ * DEEPWELL - Wikijump API provider and database manager
+ * Copyright (C) 2019-2023 Wikijump Team
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#[derive(Deserialize, Debug, Copy, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct SiteMembership {
+    pub site_id: i64,
+    pub user_id: i64,
+}
+
+#[derive(Deserialize, Debug, Copy, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct SiteMembershipHistory {
+    pub user_id: Option<i64>,
+    pub site_id: Option<i64>,
+    pub current_members: Option<bool>,
+    pub start_id: i64,
+    pub limit: u64,
+}
```

Comment:
Where is this struct used?

Reply:
It was used in the membership history methods before I removed them. I'm not sure why clippy didnt give off a warning for them.